### DISCUSSION TITLE
Problems with Stellarium 23.2

### DISCUSSION THREAD

**Discussion by Orio1961** [Stellarium/stellarium#3340] [General]
I installed the new version of Stellarium 23.2 and ran into problems:
1. I have Win 7 64bit installed on my computer, and in the new release there is no installer for this Win, only 32bit, this is very sad.
2. When installing the program, I encountered such a problem ''Runtime error!
R6034
An application has made an attempt to load the C runtime library incorrectly. Please contact the application's support team for more information.''
The program was still installed, but when updating the catalogs, the following happens: an error occurred when loading stars4, I can’t download anything else, everything was fine in previous versions, it’s a pity that I demolished it (((. Well, there are still not many small bugs, but for me the most important thing is these. Please pay attention to this and once again increase the number of star catalogs and add, for convenience, the ''Copy'' button for quick copying of the information that is available on hover to a specific object. Many thanks in advance for fixing these bugs.

**Comment by alex-w** [comment]
1. It’s not correct, please see the list of packages: https://github.com/Stellarium/stellarium/releases/tag/v23.2
2. OpenSSL issue is fixed in latest beta: https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot

**Comment by Orio1961** [comment]
Thank you very much for the information and the quick solution of the

problem, I will try again

------- Original Message -------
понедельник, 24 июля 2023 г., 21:37, Alexander V. Wolf ***@***.***> написал:
